.github/workflows/api-deploy-worker.yml

```
@@ -37,6 +37,7 @@ jobs:
       ECS_SERVICE: ${{ inputs.name}}-${{ inputs.env }}-service
       ECR_REPOSITORY: jfp-${{ inputs.name }}-${{ inputs.env }}
       IMAGE_TAG: ${{ github.sha }}
+      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
```

actionlint (1.7.7)

property "nx_cloud_access_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; aws_default_region: string; aws_network_config: string; github_token: string; hive_token: string; jfp_aws_access_key_id: string; jfp_aws_secret_access_key: string}

(expression)